README.md

```diff
@@ -1,12 +1,23 @@
-## Notice
+# :warning: **kubeflow/examples Repository is not Maintained**
```

Comment:
TIL, I was [aware of admonitions](https://github.com/orgs/community/discussions/16925#discussion-4085374) but not this option :)